apps/omg_db/lib/db.ex

```diff
@@ -29,64 +30,75 @@ defmodule OMG.DB do
 
   @type utxo_pos_db_t :: {pos_integer, non_neg_integer, non_neg_integer}
 
+  @decorate transaction_event(:LevelDB)
   def multi_update(db_updates, server_name \\ @server_name) do
     GenServer.call(server_name, {:multi_update, db_updates})
   end
 
   @spec blocks(block_to_fetch :: list(), atom) :: {:ok, list()} | {:error, any}
-  def blocks(blocks_to_fetch, server_name \\ @server_name)
-
-  def blocks([], _server_name), do: {:ok, []}
-
-  def blocks(blocks_to_fetch, server_name) do
-    GenServer.call(server_name, {:blocks, blocks_to_fetch})
+  @decorate transaction_event(:LevelDB)
+  def blocks(blocks_to_fetch, server_name \\ @server_name) do
+    case blocks_to_fetch do
+      [] -> {:ok, []}
+      blocks_to_fetch -> GenServer.call(server_name, {:blocks, blocks_to_fetch})
+    end
   end
 
+  @decorate transaction_event(:LevelDB)
```

Comment:
I'm still thinking about the naming convention for the event groups and namespaces. I think there's a problem with camel-case entries like `LevelDB` as they suggest a module, while there is no such module strictly speaking.

Perhaps we should consider using:
1/ `__MODULE__` for event groups always, consistently
2/ something more top-level for transactions' namespaces? `web`/`background` is the appsignals default, which I agree is pretty coarse, but maybe it will we what is most useful actually :thinking: ?

We also need to take AppSignal's UX into account - switching namespaces filters is very clunky, so the less the better IMHO